packages/builder/src/fee/FeeBuilder.utils.ts

```diff
@@ -0,0 +1,256 @@
+import type {
+  AnyParachain,
+  Asset,
+  ChainAssetId,
+} from '@moonbeam-network/xcm-types';
+import { isHexString } from '@moonbeam-network/xcm-utils';
+import type { ApiPromise } from '@polkadot/api';
+import type { Option, Result, u128 } from '@polkadot/types';
+import type {
+  Error as PolkadotError,
+  Weight,
+} from '@polkadot/types/interfaces';
+import type { AnyJson } from '@polkadot/types/types';
+import { XcmVersion } from '../extrinsic';
+import type { MoonbeamRuntimeXcmConfigAssetType } from './FeeBuilder.interfaces';
+
+const DEFAULT_AMOUNT = 10 ** 6;
+const DEFAULT_HEX_STRING =
+  '0xabcdef1234567890fedcba0987654321abcdef1234567890fedcba0987654321';
+
+const MOON_CHAIN_NATIVE_ASSET_ID = '0x0000000000000000000000000000000000000802';
```

Comment:
In v3 you don't need to hardcode this. You can get from a native moon chain asset.